src/go/rpk/pkg/cli/topic/produce.go

```diff
@@ -350,6 +417,24 @@ separated by an equal:
 
     %H{3}%h{ %k=%v }
 
+SCHEMA-REGISTRY
+
+Records can be encoded using a specified schema from our schema registry. Use
+the '--schema-id' or '--schema-key-id' flags to define the schema ID, rpk will
+retrieve the schemas and encode the record accordingly.
+
+Additionally, utilizing 'topic' in the mentioned flags allows for the use of the
+Topic Name Strategy. This strategy identifies a schema subject name based on the
+topic itself. For example:
+
+Produce to 'foo', encode using the latest schema in the subject 'foo-value':
+    rpk topic produce foo --schema-id=topic
+
+For protobuf schemas, you can specify the fully qualified name of the message
+you want the record to be encoded with. Use the 'schema-type' flag or
+'schema-key-type'. If the schema contains only one message, specifying the
```

Comment:
Worth having an example of how to use this flag

Reply:
Added an example in the last force push. :+1: